.github/workflows/r-unit-tests.yaml

```diff
@@ -0,0 +1,28 @@
+name: Run R Unit Tests
+
+on: [push]
+
+jobs:
+  run-tests:
+    name: R Unit Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          cache: true
```

Comment:
Is there a reason to set this manually here? If so is there a reason it isn't set everywhere else?

Reply:
They're the defaults. Removed them to match the other workflow files.